pkg/totalaccountwatcher/totalaccountwatcher.go

```diff
@@ -88,6 +89,7 @@ func (s *totalAccountWatcher) UpdateTotalAccounts(log logr.Logger) error {
 	accountTotal, err := TotalAwsAccounts()
 	if err != nil {
 		log.Error(err, "Failed to get account list with error code")
+		return nil
```

Comment:
This is goodness regardless. IIUC previously an error would cause L94 to reset the total number of accounts back down to zero :astonished:

Reply:
Wait, I take it back, this should be

```suggestion
		return err
```

so that the log on L76 happens.